### PR TITLE
Move samples from bot builder to Teams Samples Repository: JAVA

### PR DESCRIPTION
**Pull request by Mohammed-MSFT**
Raised PR for Bot Builder java samples (images readme and sample.json) is pending.

### CODE REVIEW

File: samples/bot-conversation/java/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+# Teams Conversation Bot
+
+Bot Framework v4 Conversation Bot sample for Teams.
+
+This bot has been created using [Bot Framework](https://dev.botframework.com). This sample shows
+how to incorporate basic conversational flow into a Teams application. It also illustrates a few of the Teams specific calls you can make from your bot.
+
+This sample is a Spring Boot app and uses the Azure CLI and azure-webapp Maven plugin to deploy to Azure.
```
[the remark this answers is not in the document]
Comment (ChetanSharma-msft) [reply samples/bot-conversation/java/README.md:23]: Done

File: samples/bot-conversation/java/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+## Prerequisites
+
+- Java 1.8+
+- Install [Maven](https://maven.apache.org/)
+- An account on [Azure](https://azure.microsoft.com) if you want to deploy to Azure.
+- Microsoft Teams is installed and you have an account
+- [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+
+## Concepts introduced in this sample
```
[the remark this answers is not in the document]
Comment (ChetanSharma-msft) [reply samples/bot-conversation/java/README.md]: Done

File: samples/bot-conversation/java/teamsAppManifest/manifest.json [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  "id": "<<YOUR-MICROSOFT-APP-ID>>",
+  "packageName": "com.teams.sample.teamsconversationbot",
+  "developer": {
+    "name": "teamsConversationBot",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.teams.com/privacy",
+    "termsOfUseUrl": "https://www.teams.com/termsofuser"
+  },
+  "icons": {
+    "outline": "icon-outline.png",
```
[the remark this answers is not in the document]
Comment (ChetanSharma-msft) [reply samples/bot-conversation/java/teamsAppManifest/manifest.json:14]: Done

File: samples/bot-conversation/java/README.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+![BotConversation](Images/BotConversation.gif)
+
+## To try this sample
```
[the remark this answers is not in the document]
Comment (ChetanSharma-msft) [reply samples/bot-conversation/java/README.md]: Done

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by Wajeed-msft — APPROVED** [review]
All the Setup Image for Java and Maven are duplicated. Let's keep it as part of first sample and delete it from other locations. Let's reference the same Setup.md file everywhere instead of duplicating.

**Pull request merged**